tests/FakeItEasy.Tests.Approval.netstd/ApiApproval.cs

```diff
@@ -0,0 +1,33 @@
+namespace FakeItEasy.Tests.Approval
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+    using System.Runtime.CompilerServices;
+    using ApprovalTests;
+    using ApprovalTests.Reporters;
+    using PublicApiGenerator;
+    using Xunit;
+
+    public class ApiApproval
+    {
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public void ApproveApiNetStd()
+        {
+            // Approvals and PublicApiGenerator aren't available for .NET Core, so
+            // we'll load the .NET Standard FakeItEasy assembly from disk and
+            // examine it.
+            string codeBase = Assembly.GetExecutingAssembly().CodeBase;
+            UriBuilder uri = new UriBuilder(codeBase);
+            string assemblyPath = Uri.UnescapeDataString(uri.Path);
+            var containingDirectory = Path.GetDirectoryName(assemblyPath);
+            var configurationName = new DirectoryInfo(containingDirectory).Name;
+            var assemblyFile = @"..\..\..\..\src\FakeItEasy.netstd\bin\" + configurationName + @"\FakeItEasy.dll";

```

Comment:
There may be a less brittle way, but I've not come across it yes.
Referencing seems not to work. As I understand it it's because FakeItEasy targets .NET Standard 1.6, and we'll need the test project to target .NET 4.6.3 in order to reference it.
For now, if I try I get

```
System.IO.FileNotFoundException

Could not load file or assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

   at FakeItEasy.Tests.Approval.ApiApproval.ApproveApiNetStd()
```